### ISSUE TITLE
Why does contig_id have file_id| before it?

### ISSUE THREAD

**Issue by tseemann** [phac-nml/mob-suite#11]
```
file_id cluster_id      contig_id       contig_length   circularity_status      rep_type        rep>
kleb.fa 769     kleb.fa|2018-11623.74   1036    Incomplete                                      LT0>
kleb.fa 769     kleb.fa|2018-11623.46   3671    Incomplete                                      LT0>
```
The contig ID has the `kleb.fa|` before it.  Is this necessary?

**Comment by jrober84** [comment]
This was useful for me to have the fasta file name added into the fasta header for downstream use of the plasmid fasta files. Does this cause an issue for your workflow?

**Comment by tseemann** [comment]
Well, it's not the actual `contig_id` ? The `file_id` is already in column 1 ?
And my contig_id might have `|` in it (like NCBI ones) and how will we know how to parse it?

**Comment by kbessonov1984** [comment]
@tseemann Probably we can remove the file name from the `contig_id` values for a cleaner output and easier downstream processing. The `SeqIO.parse()` appends file name to every `contig` title line  in the input fasta file creating rather detailed `contig_id` values. Thank you for bringing this up.

**Comment by kbessonov1984** [comment]
This is addressed now in the Pull Request #21  that is now merged with master branch

**Comment by tseemann** [comment]
Thanks @tseemann
